app/build.gradle

```diff
@@ -2,64 +2,93 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'realm-android'
 apply plugin: 'deploygate'
+apply plugin: 'com.github.ben-manes.versions'
+apply plugin: 'com.github.triplet.play'
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.1"
+    buildToolsVersion '26.0.2'
+    playAccountConfigs {
+        defaultAccountConfig {
+            jsonFile = file('app-publisher.json')
+        }
+    }
+    signingConfigs {
+        release {
+            storeFile file('../SunsetScrob.jks')
+            storePassword System.getenv('SUNSET_STORE_PASSWORD')
+            keyAlias "sunsetScrob"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/d98de3fb9ea0ace0e5c0241379c958a7804774f9/app/build.gradle#L20"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>